Bottom: 1e0d3d57c011d88a7a2941898e8ab8ef02d87f2d
Top:    e3eae8a115375e8e9647f0d2db5b58cd4aea0f2d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-13 15:43:00 -0500

fixup openldap



---
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index a62a8f2e..629032ad 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -151,10 +151,11 @@
       LDAP_HOSTNAME: '{{ openldap_hostname }}'
     hostname: '{{ phpldapadmin_hostname }}'
     image: '{{ phpldapadmin_image_name }}:{{ docker_image_tag }}'
-    link: '{{ openldap_container_name }}:ldap'
     name: '{{ phpldapadmin_container_name }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+        links:
+          - '{{ openldap_container_name }}:ldap'
     ports: '{{ phpldapadmin_port_args }}'
     purge_networks: true
     restart_policy: '{{ docker_restart_policy }}'
